cleanup_copr.yml

```diff
@@ -0,0 +1,17 @@
+---
+- hosts: localhost
+  gather_facts: no
+  tasks:
+    - name: 'Get Copr repositories'
+      command: "copr-cli list @theforeman"
+      register: copr_repos
+
+    - set_fact:
+        scratch_repos: "{{ copr_repos.stdout.split('\n') | select('match', 'Name') | list | replace('Name: ', '') }}"
```

Comment:
Can you use regexes or check that the line also starts with ```Name: ```? Right now it could match a description.